### DISCUSSION TITLE
希望增加支持安卓11及以上手机开发者模式的无线调试

### DISCUSSION THREAD

**Discussion by cccyncn** [eiyooooo/Easycontrol_For_Car#14] [Ideas]
希望增加支持安卓11及以上手机开发者模式的无线ADB调试

**Comment by eiyooooo** [comment]
[V1.6.0版本](https://github.com/eiyooooo/Easycontrol_For_Car/releases/tag/V1.6.0)已增加此功能

https://github.com/eiyooooo/Easycontrol_For_Car/commit/eb1ad15792e9d0424fe577521d2ada732a0614b3

**Reply by cccyncn** [reply]
感谢大佬，只是这个不知道为什么这个功能在埃安Y的车机上没法和其他设备配对，安卓8.1

**Reply by eiyooooo** [reply]
> 感谢大佬，只是这个不知道为什么这个功能在埃安Y的车机上没法和其他设备配对，安卓8.1

检查ip和端口是否正确，注意各端口的区别

如果是打开5555端口失败的话关闭页面再打开页面重试

**Reply by cccyncn** [reply]
确定没填错，用另外一台手机连是可以的，在车机上卸载重装了好多遍就是不行。。

**Reply by eiyooooo** [reply]
> 确定没填错，用另外一台手机连是可以的，在车机上卸载重装了好多遍就是不行。。

那应该是车机安卓系统可能缺少某些加密库导致无法配对

**Reply by cccyncn** [reply]
应该是，我用甲壳虫ADB助手那个配对工具也是无法配对
